polyfill/intersection-observer.js

```diff
@@ -48,6 +48,20 @@ var document = window.document;
  */
 var registry = [];
 
+/**
+ * The signal updater for cross-origin intersection. When available, it means
```

Comment:
Insteading of saying "when available" maybe say "when set" or "when not null"? To me "available" is a bit unclear.